update_formula.sh

```diff
@@ -6,7 +6,7 @@ BINARY_BASE_URL=$(echo ${BINARY_BASE_URL} | sed 's/\//\\\//g')
 VERSION=$(echo "${VERSION}" | cut -d "v" -f2)
 
 mkdir -p Formula
-cp klotho.rb.tmpl Formula/klotho.rb
+cp klotho.rb.tmpl "Formula/${OUTPUT_FILE_NAME}"
```

Comment:
I can't comment on line 3, but maybe add `-u`? (that will cause the script to fail if `OUTPUT_FILE_NAME` isn't set).

not a blocker.

Reply:
Sounds reasonable. I'll make the change.